ajax/libs/logdown.js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "logdown.js",
+  "filename": "logdown.min.js",
+  "version": "3.2.2",
+  "description": "Debug utility with markdown support that runs on browser and server",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/caiogondim/logdown"
+  },
+  "keywords": [
+    "log",
+    "markdown",
+    "debug",
+    "console",
+    "info",
+    "error",
+    "logging"
+  ],
+  "author": "Caio Gondim <me@caiogondim.com> (https://caiogondim.com)",
+  "homepage": "http://caiogondim.github.io/logdown.js/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brightcove/videojs-thumbnails.git",
```

Comment:
The commit message said use npm auto-update, but this is git auto-update config.
Which one is correct?
Also, `target` is wrong.
Please modify the correct auto-config.
Many thanks.